packages/byods/src/token-storage-adapter/index.ts

```diff
@@ -0,0 +1,77 @@
+import {OrgServiceAppAuthorization} from '../types';
+import {TokenStorageAdapter} from './types';
+
+/**
+ * An in-memory adapter for storing, listing and retrieving service app authorization tokens.
+ *
+ * @public
+ */
+export class InMemoryTokenStorageAdapter implements TokenStorageAdapter {
+  /**
+   * The local memory cache for the tokens.
+   */
+  private tokenCache: {[orgId: string]: OrgServiceAppAuthorization};
+
+  constructor(tokenCache: {[orgId: string]: OrgServiceAppAuthorization} = {}) {
+    this.tokenCache = tokenCache;
+  }
+
+  /**
+   * Method to set the token for an organization.
+   * @param orgId Organization ID
+   * @param token Respective token
+   * @example
+   * await storageAdapter.setToken('org-id', token);
+   */
+  async setToken(orgId: string, token: OrgServiceAppAuthorization): Promise<void> {
+    this.tokenCache[orgId] = token;
```

Comment:
Did not feel a need to use try-catch for this. Memory is anyways empty by default.